roles/provision-metrics-apb/tasks/main.yml

```diff
@@ -272,3 +272,18 @@
 - name: "Create prometheus secret"
   shell: "oc create -f /tmp/secret.yaml -n {{ namespace }}"
 
+# Check the containers in the grafana pod and make sure they are all ready
+- name: "Wait for all grafana containers to become ready"
+  shell: oc get pods --namespace={{ namespace }} -o jsonpath='{.items[?(@.metadata.labels.deploymentconfig=="grafana")].status.containerStatuses[?(@.ready==true)].ready}'| wc -w
```

Comment:
this is great although it would also be possible to modify the oc command like so:

```
oc get pods --namespace={{ namespace }} --selector="deploymentconfig=prometheus"
```

This would result in a much more readable `jsonpath` query